oktapam/resource_project_group.go

```diff
@@ -237,14 +235,30 @@ func resourceProjectGroupDelete(ctx context.Context, d *schema.ResourceData, m i
 	return nil
 }
 
-func createProjectGroupResourceID(project string, group string) string {
-	return fmt.Sprintf("%s|%s", project, group)
-}
-
 func parseProjectGroupResourceID(resourceId string) (string, string, error) {
-	split := strings.Split(resourceId, "|")
+	split := strings.Split(resourceId, "/")
 	if len(split) != 2 {
-		return "", "", fmt.Errorf("oktapam_project_group id must be in the format of <project name>|<group name>, received: %s", resourceId)
+		return "", "", fmt.Errorf("expected format: <project_name>/<group_name>, received: %s", resourceId)
 	}
 	return split[0], split[1], nil
 }
+
+
+func importResourceProjectGroupState(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	// d.Id() here is the last argument passed to the `terraform import RESOURCE_TYPE.RESOURCE_NAME RESOURCE_ID` command
+	// Id provided for import is in the format <project_name>/<group_name>
+	projectName, groupName, err := parseProjectGroupResourceID(d.Id())
```

Comment:
`createProjectGroupResourceID` was removed. Isn't the ID set to the UUID, not to the concatenated value?